app/scripts/models/user.js

```diff
@@ -419,6 +419,32 @@ define(function (require, exports, module) {
         });
     },
 
+    /**
+     * Complete signin verification for the account. Notifies other tabs
+     * of signin if the account has a sessionToken and verification
+     * successfully completes.
+     *
+     * @param {object} account - account to verify
+     * @param {string} code - verification code
+     * @returns {promise} - resolves with the account when complete
+     */
+    completeAccountSignIn: function (account, code) {
```

Comment:
Ugh, we already have `completeAcocuntSignUp`... `verify` would create an inconsistency.